jenkins/PerfCI_OpenShift_Deployment/Jenkinsfile

```diff
@@ -0,0 +1,114 @@
+pipeline {
+    agent {
+        label 'PerfCi'
+    }
+    environment {
+        CONTACT_EMAIL1 = credentials('perfci_contact_email1')
+        PROVISION_PRIVATE_KEY_FILE = credentials('perfci_provision_private_key_file')
+        PROVISION_IP = credentials('perfci_provision_ip')
+        PROVISION_USER = credentials('perfci_provision_user')
+        WORKSPACE = '/home/jenkins'
+        INSTALL_OCP_VERSION = 'latest-4.14'
+        OCP_CLIENT_VERSION = '4.14.1'
+        PROVISION_PORT = 22
+        KUBEADMIN_PASSWORD_PATH = '/root/.kube/kubeadmin-password'
+        KUBECONFIG_PATH = '/root/.kube/config'
+        PROVISION_INSTALLER_PATH = '/root/jetlag/./run_jetlag.sh'
+        PROVISION_INSTALLER_CMD = 'pushd /root/jetlag;/root/jetlag/./run_jetlag.sh 1>/dev/null 2>&1;popd'
+        PROVISION_INSTALLER_LOG = 'tail -40 /root/jetlag/jetlag.log'
+        PRIVATE_KEY_PATH = '/home/jenkins/.ssh/provision_private_key'
+        CONFIG_PATH = '/home/jenkins/.ssh/config'
+        CONTAINER_PRIVATE_KEY_PATH = '/root/.ssh/provision_private_key'
+        CONTAINER_CONFIG_PATH = '/root/.ssh/config'
+        PROVISION_TIMEOUT = 7200
+    }
+    stages {
+        stage('Cleanup') {
+            steps {
+                script {
+                    sh 'sudo podman ps -a --filter "name=benchmark-runner" -q | xargs -r podman rm'
```

Comment:
That won't run `xargs -r podman rm` privileged; the `sudo` will only apply to the pipeline component it references.

Reply:
Must add sudo because Jenkins script run with jenkins user,
Got the following error without sudo: 
`ERRO[0000] XDG_RUNTIME_DIR directory "/run/user/0" is not owned by the current user`

Reply:
I understand you need to use sudo here; I think you probably also need to use `sudo xargs -r podman rm`

Reply:
Fixed